_includes/utilities/output-control.html

```diff
@@ -1,6 +1,6 @@
 <section class="utilities-section margin-top-6" id="output-control">
   <h3 class="utilities-section-title">Output control</h3>
-  <p class="utility-text">Control whether or not to output any USWDS utility family by setting the value of the <code>$[utility_family]-settings.output</code> variable found in <code>_uswds-theme-utilities.scss</code>.</p>
+  <p class="utility-text">Control whether or not to output any USWDS utility family by setting the value of the <code>$[utility_family]-settings.output</code> variable in your <a href="{{ site.baseurl }}/documentation/settings" class="">USWDS settings configuration</a>.</p>
```

Comment:
I am wondering if users will get confused by `$[utility_family]-settings.output` since you cannot actually use `display-settings.output` in the settings configuration without getting an error. It might be clearer if we provided an example that shows that `.output` represents a nested setting. Maybe something like this?



```suggestion
  <p class="utility-text">Control whether or not to output any USWDS utility family by setting the value of the <code>$[utility_family]-settings.output</code> variable in your <a href="{{ site.baseurl }}/documentation/settings" class="">USWDS settings configuration</a>. This code sample shows a how to exclude the display utilities from your CSS:</p>
  <div class="example border-left-05 border-secondary-light padding-left-105 margin-top-2">
    <pre class=" font-mono-xs margin-0 padding-0 bg-transparent">
@use "uswds-core" with (
  $display-settings: (
    output: false,
  ),
);
    </pre>
  </div>
```

![image](https://user-images.githubusercontent.com/93996430/214437250-8fc61fa5-12ab-48f4-ac6f-92f9c18591bc.png)

Reply:
I think provides a lot of extra clarity that people might miss unless they read and understand that this is a map from the settings page.

Should I try to track down other examples of maps and provide similar clarity? Take breakpoints for example

![image](https://user-images.githubusercontent.com/25211650/214602106-0fbd780f-8463-4665-ab22-9f166eda78ea.png)

Reply:
I think so. I like to err on the side of too much clarity rather than not enough. I suspect users might look only at the section they care about and would miss guidance for the other maps.